ister.py

```diff
@@ -414,20 +414,11 @@ def copy_os(args, template, target_dir):
     """
     LOG.info("Starting swupd. May take several minutes")
     add_bundles(template, target_dir)
-    swupd_command = "swupd verify --install --path={0} " \
-                    "--manifest={1}".format(target_dir, template["Version"])
-    if shutil.which("stdbuf"):
-        swupd_command = "stdbuf -o 0 {0}".format(swupd_command)
-    if args.contenturl:
-        swupd_command += " --contenturl={0}".format(args.contenturl)
-    if args.versionurl:
-        swupd_command += " --versionurl={0}".format(args.versionurl)
-    swupd_command += " --format={0}".format(args.format)
+
+    add_bundles(template, target_dir)
```

Comment:
This is happening twice now? Seems like a mistake.

Reply:
Only once, just for swupd